crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -133,9 +151,15 @@ impl ParsableValue for SuiExtraValueArgs {
     }
 
     fn concrete_vector(elems: Vec<Self::ConcreteValue>) -> anyhow::Result<Self::ConcreteValue> {
-        Ok(SuiValue::MoveValue(MoveValue::Vector(
-            elems.into_iter().map(SuiValue::assert_move_value).collect(),
-        )))
+        if !elems.is_empty() && matches!(elems[0], SuiValue::Object(_)) {
```

Comment:
ah interesting. Can we not test an empty `ObjVec` then?

Reply:
We cannot. Since (I believe) we don't know the context in which the argument is used, we have to decide to either represent empty vectors as a primitive type or as an empty vectors of objects (which are different `CallArg`s).